### DISCUSSION TITLE
animation to complement x-show

### DISCUSSION THREAD

**Discussion by anjanesh** [alpinejs/alpine#3688] [1. Help] [answered]
This works - but I want the row deleted slowly (animation-wise) when the button is clicked.
```
<tr x-show="!rule.thrashed">
<td>data</td>
<td>    
    <button type="button" @click="rule.thrashed = true">Delete</button>
</td>
</tr>
```
Is there a way to add this animation to the `x-show` when `thrashed`  is true ?

**Comment by ekwoka** [comment, marked as answer]
https://alpinejs.dev/directives/transition
